src/quartic/common/utils.py

```diff
@@ -69,14 +69,11 @@ def load_package(package_dir):
         yield m
 
 def load_modules(package_dirs):
+    DslContext.reset()
     for package_dir in package_dirs:
         for m in load_package(package_dir):
             yield m
 
 def get_pipeline_from_args(dirs):
-    steps = []
-    for module in load_modules(dirs):
-        for _, v in module.__dict__.items():
-            if isinstance(v, Step):
-                steps.append(v)
-    return steps
+    list(load_modules(dirs))
```

Comment:
Is there a clearer way to express the idea that we just want to evaluate this generator til the end?